PCbuild/pythoncore.vcxproj

```diff
@@ -651,7 +651,11 @@
     <ClCompile Include="..\Python\structmember.c" />
     <ClCompile Include="..\Python\symtable.c" />
     <ClCompile Include="..\Python\sysmodule.c">
-      <PreprocessorDefinitions>VPATH="$(PyVPath)";%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>
+        VARCH_NAME="$(ArchName)";
+        VPATH="$(PyVPath)";
+        %(PreprocessorDefinitions)
+      </PreprocessorDefinitions>
```

Comment:
@zooba: Is there a reason why definitions are on a single line? Should it be reverted to a single line?

Reply:
I think this is more readable. In fact they can be described in one line.